frontend/src/summary.pug

```diff
@@ -72,7 +72,7 @@
     .error-message-box__close-button(onclick="dismissTab(this)") &times;
     .error-message-box__message The following issues occurred when analyzing the following repositories:
     .error-message-box__failed-repo(v-for="errorBlock in errorMessages")
-      i.fas.fa-exclamation.mini-font
+      i.fas.fa-exclamation
```

Comment:
Seems like you did not include the mixin for this

Reply:
Ya seemed to have missed this.